Tests/modules/io_related/test_codecs.py

```diff
@@ -390,12 +404,27 @@ def test_cp11334(self):
 
         self.assertEqual(len(t_out_lines), 0)
         self.assertTrue(t_err_lines[0].startswith(b"  File"))
+        self.assertTrue(t_err_lines[1].startswith(b"SyntaxError: Non-UTF-8 code starting with '\\xb5' in file"))
+
+        #--Test that using "# coding ..." is OK
+        p = subprocess.Popen([sys.executable, os.path.join(self.test_dir, "encoded_files", "cp11334_ok.py")], shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        t_in, t_out, t_err = (p.stdin, p.stdout, p.stderr)
+        t_err_lines = t_err.readlines()
+        t_out_lines = t_out.readlines()
+        t_err.close()
+        t_out.close()
+        t_in.close()
+
+        self.assertEqual(len(t_err_lines), 0)
         if is_cli:
-            self.assertTrue(t_err_lines[1].startswith(b"SyntaxError: Non-ASCII character '\\xb5' in file"))
+            print("CodePlex 11334")
+            self.assertEqual(t_out_lines[0].rstrip(), b"\xe6ble")
         else:
-            self.assertTrue(t_err_lines[1].startswith(b"SyntaxError: Non-UTF-8 code starting with '\\xb5' in file"))
+            self.assertEqual(t_out_lines[0].rstrip(), b"\xb5ble")
+        self.assertEqual(len(t_out_lines), 1)
 
-        #--Test that using "# coding ..." is OK
+        #--Test that using other whitespace characters (rather than [ \t\f]) is OK
+        #but non-ASCII letters in the encoding name are not accepted
         p = subprocess.Popen([sys.executable, os.path.join(self.test_dir, "encoded_files", "cp11334_ok.py")], shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Shouldn't this test be using `cp11334_ok2.py`?

Reply:
right